docs/specs/om/open_metrics_spec_2_0.md

```diff
@@ -749,20 +759,19 @@ An example of a Metric with no labels and a MetricPoint with Sum, Count, and Cre
 
 ```
 # TYPE foo histogram
-foo_bucket{le="0.0"} 0
-foo_bucket{le="1e-05"} 0
-foo_bucket{le="0.0001"} 5
-foo_bucket{le="0.1"} 8
-foo_bucket{le="1.0"} 10
-foo_bucket{le="10.0"} 11
-foo_bucket{le="100000.0"} 11
-foo_bucket{le="1e+06"} 15
-foo_bucket{le="1e+23"} 16
-foo_bucket{le="1.1e+23"} 17
-foo_bucket{le="+Inf"} 17
-foo_count 17
-foo_sum 324789.3
-foo_created 1520430000.123
+foo_bucket{le="0.0"} 0 ct@1520430000.123
```

Comment:
Same comment about created timestamps all being different.  Count and sum should match the lowest CT